Objects/funcobject.c

```diff
@@ -222,7 +216,25 @@ func_set_dict(PyFunctionObject *op, PyObject *value)
 }
 
 static PyObject *
-func_get_code(PyFunctionObject *op)
+func_get_dict(PyFunctionObject *op)
+{
+     if (PyErr_WarnPy3k_WithFix("The attribute func_dict is not supported in 3.x, ",
+                       "use '__dict__' instead", 1) < 0)
+        return -1;
+    return get_dict(op);
+}
+
+static int
+func_set_dict(PyFunctionObject *op, PyObject *value)
+{
+     if (PyErr_WarnPy3k_WithFix("The attribute func_dict is not supported in 3.x, ",
```

Comment:
It looks like the indentation has gone a bit wonky here? [`if` is further indented than `return`.]

Reply:
I addressed this one, you can review other areas too.

Reply:
My comment was even more shallow! The `if` on line 230 (and line 281 and others below) is 5 spaces rather than the standard 4. I know I'm nit-picking!

Reply:
OK